hack empty answer key sociology/if statement in the bake.rb sociology file

```diff
@@ -0,0 +1,9 @@
+if book.search('.os-eob.os-solution-container[data-type = "composite-chapter"]').first.contains?(' > [data-type="composite-page"] > div[data-type="solution"]')
+    book.search('.os-eob.os-solution-container[data-type = "composite-chapter"]').first.add_class('has-solutions')
+    book.search('body').first.add_class('has-answerkey')
+  else
+    book.search('.os-eob.os-solution-container[data-type = "composite-chapter"]').first.add_class('no-solutions')
+    book.search('.os-eob.os-solution-container.no-solutions[data-type = "composite-chapter"]').first.trash
+    book.search('body').first.add_class('no-answerkey')
+    book.search('body.no-answerkey > #toc > ol > li.os-toc-composite-chapter').trash
+  end
```

Comment:
I have written some similar hacky solution to adjust the sociology recipe so it can be used by both 2e and 3e editions as BrritanyJ have added within this PR in cnx-recipes repo: https://github.com/openstax/cnx-recipes/pull/3829. @brittany-johnson please let me know what are your thougts about writing it this way in Kitchen and if we need something less hacky and which can be reused in other books I can write some direction/module based on it. cc @jpslav

Reply:
@ajluk Hmm.. I think instead we should do something different to address this. 

I believe there are two directions for the answer key - one for the container `BookAnswerKeyContainer` and the strategies in `MoveSolutionsToAnswerKey`.

One idea is to add a conditional in sociology recipe to only call those two directions if solutions are found in the book passed to bake. 
I think this could work for both 2e & 3e. 

What do you think? @ajluk

Reply:
@brittany-johnson Thank you! I have added the changes you suggested in the `recipes` repo to sociology recipe PR directly: https://github.com/openstax/recipes/pull/24. Let me know if this is similar to something you have in mind? :slightly_smiling_face:

Reply:
Yes! This is what I had in mind - thanks for adding it. 
This file can be deleted now